src/styles/blocks/variable.scss

```diff
@@ -0,0 +1,3 @@
+$sizeicon: 40px;
```

Comment:
It's a good practice to use more descriptive variable names. Consider changing $sizeicon to $icon-size.